zeppelin-server/src/test/java/org/apache/zeppelin/rest/NotebookRestApiTest.java

```diff
@@ -78,6 +78,39 @@ public void setUp() {
     anonymous = new AuthenticationInfo("anonymous");
   }
 
+  @Test
+  public void testGetReloadNote() throws IOException {
+    LOG.info("Running testGetNote");
+    Note note1 = null;
+    try {
+      note1 = TestUtils.getInstance(Notebook.class).createNote("note1", anonymous);
+      note1.addNewParagraph(AuthenticationInfo.ANONYMOUS);
+      TestUtils.getInstance(Notebook.class).saveNote(note1, anonymous);
+      CloseableHttpResponse get = httpGet("/notebook/" + note1.getId());
+      assertThat(get, isAllowed());
+      Map<String, Object> resp = gson.fromJson(EntityUtils.toString(get.getEntity(), StandardCharsets.UTF_8),
+              new TypeToken<Map<String, Object>>() {}.getType());
```

Comment:
That's right, there're many places in test that needs to do code refactoring, this could be done in a separated PR. If you would like to make contribution in Zeppelin, you can start with that I believe.